pkg/commands/iamcleanup/command.go

```diff
@@ -0,0 +1,133 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package iamcleanup
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/abcxyz/pkg/cli"
+	"github.com/abcxyz/pkg/logging"
+
+	"github.com/abcxyz/guardian/internal/version"
+)
+
+var _ cli.Command = (*IAMCleanupCommand)(nil)
+
+type IAMCleanupCommand struct {
+	cli.BaseCommand
+
+	// testFlagSetOpts is only used for testing.
+	testFlagSetOpts []cli.Option
+
+	flagScope                 string
+	flagIAMQuery              string
+	flagEvaluateCondition     bool
+	flagMaxConcurrentRequests int64
+	flagWorkingDirectory      string
+}
+
+func (c *IAMCleanupCommand) Desc() string {
+	return `Detect IAM drift in a GCP organization`
+}
+
+func (c *IAMCleanupCommand) Help() string {
+	return `
+Usage: {{ COMMAND }} [options]
+
+  Detect IAM drift in a GCP organization.
+`
+}
+
+func (c *IAMCleanupCommand) Flags() *cli.FlagSet {
+	set := cli.NewFlagSet(c.testFlagSetOpts...)
+
+	// Command options
+	f := set.NewSection("COMMAND OPTIONS")
+
+	f.StringVar(&cli.StringVar{
+		Name:    "scope",
+		Target:  &c.flagScope,
+		Example: "123435456456",
+		Usage: `The scope to cleanup IAM for - organizations/123456 will cleanup all 
+		IAM matching your query in the organization and all folders and projects beneath it.`,
+	})
+	f.StringVar(&cli.StringVar{
+		Name:    "iam-query",
+		Target:  &c.flagIAMQuery,
+		Example: "policy:abcxyz-aod-expiry",
+		Usage:   `The query to use to filter on IAM.`,
+	})
+	f.BoolVar(&cli.BoolVar{
+		Name:    "evaluate-condition",
+		Target:  &c.flagEvaluateCondition,
+		Example: "false",
+		Default: true,
+		Usage: `Whether or not to evaluate the IAM Condition Expression and only delete
+		those IAM with false evaluation. Defaults to true.
+		Example: An IAM condition with expression 'request.time < timestamp("2019-01-01T00:00:00Z")'
+		will evaluate to false and the IAM will be deleted.`,
+	})
+	f.Int64Var(&cli.Int64Var{
+		Name:    "max-conncurrent-requests",
+		Target:  &c.flagMaxConcurrentRequests,
+		Example: "10",
```

Comment:
Examples should differ from defaults: https://github.com/abcxyz/readability/wiki/Go-Best-Practices#command-line-tools-clis

Reply:
Done